src/routes/herokuauth.routes.js

```diff
@@ -55,6 +55,31 @@ router.post(
   herokuAuthAuthController.addHerokuTokenForUser
 );
 
+/**
+ * @api {POST} /auth/heroku2 Exchange Heroku code for Heroku token
+ * @apiVersion 1.0.0
+ * @apiName Exchange for Heroku Authentication Token
+ * @apiGroup Heroku
```

Comment:
Should the GitHub and Heroku routes both be in an `Auth` apiGroup instead of separate groups? That way they appear under the same section in the api docs.

Reply:
I think it is really up to us. However, these endpoints would describe how to interact with the services, which is why I originally separated them.